cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1519,335 +1519,372 @@ function Start_Containerd {
   Log-Output "Starting containerd service"
   Start-Service containerd
 }
-
-# TODO(pjh): move the Stackdriver logging agent code below into a separate
+# TODO(pjh): move the logging agent code below into a separate
 # module; it was put here temporarily to avoid disrupting the file layout in
 # the K8s release machinery.
-$STACKDRIVER_VERSION = 'v1-11'
-$STACKDRIVER_ROOT = 'C:\Program Files (x86)\Stackdriver'
+$LOGGINGAGENT_VERSION = 'v1-11'
+$LOGGINGAGENT_ROOT = 'C:\fluent-bit'
+$LOGGINGAGENT_SERVICE = 'fluent-bit'
+$LOGGINGAGENT_CMDLINE = '*fluent-bit.exe*'
 
-# Restarts the Stackdriver logging agent, or starts it if it is not currently
-# running. A standard `Restart-Service StackdriverLogging` may fail because
-# StackdriverLogging sometimes is unstoppable, so this function works around it
-# by killing the processes.
+$LOGGINGEXPORTER_ROOT = 'C:\flb-exporter'
+$LOGGINGEXPORTER_SERVICE = 'flb-exporter'
+$LOGGINGEXPORTER_BINARY = 'C:\flb-exporter\flb-exporter.exe'
+$LOGGINGEXPORTER_CMDLINE = '*flb-exporter.exe*'
+
+# Restart Logging agent
 function Restart-LoggingAgent {
-  Stop-Service -NoWait -ErrorAction Ignore StackdriverLogging
+   Restart-LogService $LOGGINGEXPORTER_SERVICE $LOGGINGEXPORTER_CMDLINE
+   Restart-LogService $LOGGINGAGENT_SERVICE $LOGGINGAGENT_CMDLINE
+}
+
+# Restarts the service, or starts it if it is not currently
+# running. A standard `Restart-Service` may fail because
+# the process is sometimes unstoppable, so this function works around it
+# by killing the processes.
```

Comment:
With fluent-bit and flb-exporter now, should we try to remove all this extra restart logic that we needed for the unreliable Stackdriver agent?

Reply:
I see the same behavior with fluentbit also. The process needed to be killed. Iam sure they will fix it at some point but for now it is the case.

Reply:
Ok, thanks for checking.